README.md

```diff
@@ -66,6 +66,17 @@ Keybinding | Description                                                    |
 
 See `rspec-mode.el` for further usage.
 
+### Before verification hook
```

Comment:
If we need to have the hooks mentioned in README, let's use just one section for them. And please replace the use of dash (-) with parentheses.